runbotics/runbotics-orchestrator-ui/src/src-app/store/slices/ProcessInstance/ProcessInstance.state.ts

```diff
@@ -14,11 +14,17 @@ export interface ProcessInstanceState {
         byId: Record<string, IProcessInstance>;
         byProcessId: Record<number, IProcessInstance[]>;
         ids: string[];
-        page: Page<IProcessInstance> | null;
+        page: Page<InstanceExtendedWithSubProcesses> | null;
     };
 }
 
 export interface ProcessInstanceRequestCriteria extends Omit<IProcessInstance, 'process' | 'bot'> {
     botId?: number;
     processId?: number;
 }
+
+export interface InstanceExtendedWithSubProcesses extends IProcessInstance {
```

Comment:
subprocess is a single word, no need for upper case

Reply:
as you wish, 70 occurrences changed